console.go

```diff
@@ -260,6 +239,10 @@ func (w ConsoleWriter) writeFields(evt map[string]interface{}, buf *bytes.Buffer
 			}
 		}
 
+		// Write space only if something has already been written to the buffer
```

Comment:
Moved to inside the loop to eliminate the duplication of before loop and end if loop below.